src/test/resources/initial-dataset.xml

```diff
@@ -11,51 +11,69 @@ the table, and second identifies the item. This minimizes possible confusion of
     <auth_user id="-3" password="none" last_login="2013-02-26 14:17:19.0" is_superuser="false" username="mage" first_name="Message" last_name="Mage" email="code@nyaruka.com" is_staff="false" is_active="true" date_joined="2013-02-26 14:17:19.0" />
 
     <!-- Org #1 with a valid twilio config, anon=false -->
-    <orgs_org id="-11" is_active="true" created_by_id="-2" created_on="2013-08-09 14:43:46.925" modified_by_id="-2" modified_on="2013-08-10 15:11:26.372" name="Test org #1" msg_last_viewed="2013-08-09 14:43:46.925" webhook_events="0" plan="FREE" plan_start="2013-08-09 14:43:46.925" timezone="Africa/Johannesburg" flows_last_viewed="2013-08-10 15:11:26.370" date_format="D" config="{&quot;ACCOUNT_SID&quot;:&quot;ACe0da56e18798f1bdd9569def026e2ee7&quot;,&quot;ACCOUNT_TOKEN&quot;:&quot;6abb5cfa9f8b9018516cb6159088ef8d&quot;}" is_anon="false" />
+    <orgs_org id="-11" is_active="true" created_by_id="-2" created_on="2013-08-09 14:43:46.925" modified_by_id="-2" modified_on="2013-08-10 15:11:26.372" name="Test org #1" msg_last_viewed="2013-08-09 14:43:46.925" webhook_events="0" plan="FREE" plan_start="2013-08-09 14:43:46.925" timezone="Africa/Johannesburg" flows_last_viewed="2013-08-10 15:11:26.370" date_format="D" brand="test.com" config="{&quot;ACCOUNT_SID&quot;:&quot;ACe0da56e18798f1bdd9569def026e2ee7&quot;,&quot;ACCOUNT_TOKEN&quot;:&quot;6abb5cfa9f8b9018516cb6159088ef8d&quot;}" is_anon="false" />
+    <msgs_systemlabel id="-11" org_id="-11" label_type="I" count="0" />
```

Comment:
It's necessary to add something to this table after orgs_org inserts, so that dbunit deletes the tables in the correct order (the reverse of how it finds them in this file)